.ci/prow/openshift-test.sh

```diff
@@ -4,11 +4,51 @@ set -e #fail in case of non zero return
 
 CI_TEST=${CI_TEST:-pulp}
 API_ROOT=${API_ROOT:-"/pulp/"}
+BC_NAME="pulp-operator"
+
+echo "Creating build config $BC_NAME"
+oc new-build --strategy docker --binary --image quay.io/operator-framework/ansible-operator:v1.22.1 --name $BC_NAME
+
+echo "Waiting bc $BC_NAME sync repo ..."
+while true ; do if [ $(oc get bc $BC_NAME -oname) ] ; then break ; else sleep 5 ; fi ; done
+
+echo "Starting to build container image ..."
+oc start-build $BC_NAME --from-dir . --follow
+
+# wait a little bit to the build update its status (sometimes the --follow returns but the build .status.phase was not yet updated)
+while [ $(oc get build ${BC_NAME}-$(oc get bc $BC_NAME -ojsonpath='{.status.lastVersion}')  -ojsonpath='{.status.phase}') == 'Running' ] ; do sleep 2 done
+
+echo "Checking if build completed ..."
+if [ $(oc get build ${BC_NAME}-$(oc get bc $BC_NAME -ojsonpath='{.status.lastVersion}')  -ojsonpath='{.status.phase}') != 'Complete' ] ; then
+  echo "Build failed!"
+  exit 1
+fi
```

Comment:
what are you building here?

Reply:
Our pulp-operator image.

This is just a workaround to avoid having to open a new PR in `openshift/release` to update our config to build the image through `ci-operator` (which I believe should be the right way to do it).